11.compound_ranking_comparison.ipynb

```diff
@@ -50,32 +50,30 @@
    "source": [
     "def get_disease_df(disease_id, db_password, supervised_predictions_df):\n",
     "    # TODO ensure this query works in general\n",
-    "    query = f'''SELECT outer_pc.dwpc as dwpc, outer_pc.p_value as p_value, outer_pc.metapath_id as metapath_id, \n",
-    "                      top_ids.source_name as source_name, top_ids.target_name as target_name \n",
-    "    FROM \n",
-    "        (SELECT dwpc, p_value, metapath_id, source_id, target_id, n1.name AS source_name, n2.name AS target_name \n",
-    "         FROM dj_hetmech_app_pathcount pc \n",
-    "         JOIN dj_hetmech_app_node join_node  \n",
-    "             ON pc.target_id=join_node.id OR pc.source_id=join_node.id \n",
-    "         JOIN dj_hetmech_app_node n1 \n",
-    "             ON pc.source_id = n1.id \n",
-    "         JOIN dj_hetmech_app_node n2 \n",
-    "             ON pc.target_id = n2.id     \n",
-    "         WHERE join_node.identifier='{disease_id}' \n",
-    "         ORDER BY pc.p_value) AS top_ids \n",
-    "    JOIN dj_hetmech_app_pathcount outer_pc \n",
-    "         ON (top_ids.source_id = outer_pc.source_id AND \n",
-    "             top_ids.target_id = outer_pc.target_id) OR \n",
-    "             (top_ids.source_id = outer_pc.target_id AND \n",
-    "             top_ids.target_id = outer_pc.source_id)\n",
-    "    ORDER BY outer_pc.p_value;\n",
+    "    query = f'''SELECT dwpc, p_value, metapath_id, source_node.name AS source_name, \n",
```

Comment:
Would be great to improve formatting so the query all has the same starting indentation. Could split query out of fxn.